File: C/unit_4/Dd.c
#include<stdio.h>
#include<math.h>
int main()
{
  int x,y,z;
  printf("Enter the 1st side of the Traingle : \n");
  scanf("%d",&x);
  printf("Enter the 2nd side of the Traingle : \n");
  scanf("%d",&y);
  printf("Enter the 3rd side of the Traingle : \n");
  scanf("%d",&z);
  if(x==y&&y==z&&z==x)
    printf("Traingle is Equiateral !!!");
  else if(pow(x,2)+pow(y,2)==pow(z,2)||pow(x,2)+pow(z,2)==pow(y,2)||pow(y,2)+pow(z,2)==pow(x,2))
    printf("Traingle is Right Angled !!!");
  else
    printf("Traingle is Isoceles !!!");
  return 0;
}

File: Java/unit_4/Dd.java
package Java.unit_4;

import java.util.Scanner;

public class Dd {
    public static void main(String[] args) {
        int x, y, z;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the 1st side of the Traingle : \n");
        x = sc.nextInt();
        System.out.println("Enter the 2nd side of the Traingle : \n");
        y = sc.nextInt();
        System.out.println("Enter the 3rd side of the Traingle : \n");
        z = sc.nextInt();
        if (x == y && y == z && z == x)
            System.out.println("Traingle is Equilateral !!!");
        if (x == y || y == z || z == x)
            System.out.println("Traingle is Isoceles !!!");
        if (Math.pow(x, 2) + Math.pow(y, 2) == Math.pow(z, 2) || Math.pow(x, 2) + Math.pow(z, 2) == Math.pow(y, 2)
                || Math.pow(y, 2) + Math.pow(z, 2) == Math.pow(x, 2))
            System.out.println("Traingle is Right Angled !!!");
        sc.close();
    }
}
